transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DeserializationFailedException.java

```diff
@@ -0,0 +1,8 @@
+package com.gruelbox.transactionoutbox;
+
+/** Thrown when de-serialization of an invocation fails. */
+class DeserializationFailedException extends RuntimeException {
```

Comment:
I think there's a standard exception somewhere in the core lib for generically wrapping checked exceptions, which is all that's going on here.

Reply:
Thanks for the hint. Done.